kong/templates/nginx_kong.lua

```diff
@@ -94,6 +99,9 @@ server {
         proxy_set_header X-Forwarded-Proto $scheme;
         proxy_set_header Host $upstream_host;
         proxy_pass_header Server;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection $connection_upgrade;
```

Comment:
Not familiar enough with this to see whether it would have any side effects... @thibaultcha ?